src/core/server/saved_objects/export/get_sorted_objects_for_export.ts

```diff
@@ -109,7 +109,7 @@ async function fetchObjectsToExport({
       type: types,
       search,
       perPage: exportSizeLimit,
-      namespace,
+      namespaces: namespace ? [namespace] : undefined,
```

Comment:
We'd have difficulties exporting from multiple namespaces if there was an ID collision. For example, exporting both `marketing:dashboard:foo` and `sales:dashboard:foo` would give you two objects in the export with the same id. Since exports don't include the namespaces, there wouldn't be a way to distinguish them from each other.